File: javascript/languageTools/differentNames.js
let str='Abhishek harshit Ayush harshit Ayush Iti Deepak Ayush Iti Iti';
let arr=str.trim().split(" ");
let checked='';
let c=1;
for(let i=0;i<arr.length;i++){
    for(let j=i+1;j<arr.length;j++){
        if(arr[i]===arr[j]){
            c++;

        }
    }
    if(c>1){
        if(arr[i]!=checked){
            console.log(arr[i],c);
            checked=arr[i]

        }
        c=1;

    }
}

File: python/language_tools/different_names.py
str='Abhishek harshit Ayush harshit Ayush Iti Deepak Ayush Iti Iti'
process_string=str+' '
word=''
li = list(str.split(" "))
c=0
checked=''
def check(word):
    global c,checked
    for a in li:
        if(a==word):
            c+=1
        if(c>1):
            if(word!=checked):
                print(word,c)
                checked=word
        c=0

for s in str:
    if(s!=' '):
        word+=s
    else:
        check(word)
        word=''
